moban/plugins/template.py

```diff
@@ -180,7 +183,8 @@ def expand_template_directories(dirs):
 def expand_template_directory(directory):
     log.debug("Expanding %s..." % directory)
     translated_directory = None
-    if ":" in directory and directory[1] != ":":
+    if ":" in directory and _is_windows_drive_notation(directory) is False:
+        _assert_correct_grammer(directory)
         library_or_repo_name, relative_path = directory.split(":")
```

Comment:
this should be `split(':', 1)`

fixing only this line will mean that the exception shown to the user is more helpful for the user.

Reply:
_assert_correct_grammer(directory) will do the error handling. still, I can optimize directory.split(":")